apps/dot-cdn/src/app/dotcdn.store.ts

```diff
@@ -0,0 +1,176 @@
+     * @memberof DotCDNStore
+     */
+    readonly getChartStats = this.effect((period$: Observable<string>) => {
+        return period$.pipe(
+            mergeMap((period) => {
+                // Dispatch the loading state
+                this.dispatchLoading({
+                    loadingState: LoadingState.LOADING,
+                    loader: Loader.CHART
+                });
+                return this.dotCdnService.requestStats(period).pipe(
+                    tap({
+                        next: (data: ResponseView<DotCDNStats>) => {
+                            // Now the chart is loaded
+                            this.dispatchLoading({
+                                loadingState: LoadingState.LOADED,
+                                loader: Loader.CHART
+                            });
+
+                            const stats: DotCDNStats = data.pick('stats');
+
+                            const chartData: ChartData = {
+                                labels: this.getLabels(stats.bandwidthUsedChart),
+                                datasets: [
+                                    {
+                                        label: 'Bandwidth Used',
+                                        data: Object.values(stats.bandwidthUsedChart),
+                                        borderColor: '#42A5F5',
+                                        fill: false
+                                    }
+                                ]
+                            };
+
+                            const statsData: DotChartStats[] = [
+                                {
+                                    label: 'Bandwidth Used',
+                                    value: stats.bandwidthPretty,
+                                    icon: 'insert_chart_outlined'
+                                },
+                                {
+                                    label: 'Requests Served',
+                                    value: `${stats.totalRequestsServed}`,
+                                    icon: 'file_download'
+                                },
+                                {
+                                    label: 'Cache Hit Rate',
+                                    value: `${stats.cacheHitRate.toFixed(2)}%`,
+                                    icon: 'file_download'
+                                }
+                            ];
+
+                            this.addChartData(chartData);
+                            this.addStatsData(statsData);
+                        },
+                        error: (e) => {
+                            // TODO: Handle error
+                            console.log(e);
+                        }
+                    }),
+                    catchError(() => EMPTY)
+                );
+            })
+        );
+    });
+
+    private getLabels(data: { [key: string]: number }): string[] {
+        return Object.keys(data).map((label) => label.split('T')[0]);
+    }
```

Comment:
I'm guessing this is for the response but for the sake of future devs, please put a comment on what this transform is doing.